src/index.jsx

```diff
@@ -24,10 +22,7 @@ const reducer = combineReducers({
 
 const middleware = [thunk];
 /* eslint-disable no-underscore-dangle */
-const store = createStore(reducer, compose(
-  applyMiddleware(...middleware),
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
-));

```

Comment:
@thomHayner @jwhooperjr Mike and I determined that this was what was causing the strange errors Mike was seeing. We should be able to use the react devtools to help debug redux stuff as well